src/core/CompUnit/Repository/Installation.pm

```diff
@@ -164,25 +168,25 @@ sub MAIN(:$name is copy, :$auth, :$ver, *@, *%) {
                 }
             }
         }
```

Comment:
What's missing is something like:
if ($version < 2) {
     # upgrade the files in dist/ to include the "resources/" path part in the resource paths.
     # I guess we can check if they exist in $resources-dir to know which ones to change
}
Otherwise we end up with:
nine@sphinx:~> perl6 -e 'use lib:from<Perl5>, "lib";'
Use of uninitialized value <element> of type Any in string context
Any of .^name, .perl, .gist, or .say can stringify undefined things, if needed.  in block  at /home/nine/rakudo/install/share/perl6/site/sources/3B0B691884D053BE76F42DB8F36C3421FCE9FB4C (Inline::Perl5) line 18
===SORRY!=== Error while compiling /home/nine/site#sources/3B0B691884D053BE76F42DB8F36C3421FCE9FB4C (Inline::Perl5)
An exception occurred while evaluating a constant
at /home/nine/site#sources/3B0B691884D053BE76F42DB8F36C3421FCE9FB4C (Inline::Perl5):56
Exception details:
  ===SORRY!=== Error while compiling 
  Cannot locate native library '/home/nine/rakudo/install/share/perl6/site/libresources.so': /home/nine/rakudo/install/share/perl6/site/libresources.so: cannot open shared object file: No such file or directory

We're also gonna need an equivalent compatibility scheme in method resource because the repo-upgrade will only run on installation of a new distribution. Until that is done the repo will stay at version 1 (or even 0) and we won't find those files.